insights/parsers/up2date_log.py

```diff
@@ -0,0 +1,29 @@
+from .. import LogFileOutput, parser
```

Comment:
Please add the corresponding docstring file header. Refer to the `insights/parsers/httpd_V.py` (rebase to the latest master please)

Reply:
Also please add a `.rst` file to include this module in the documentation as described here: http://insights-core.readthedocs.io/en/latest/docs_guidelines.html#documentation-guidelines